etc/scripts/release.hs

```diff
@@ -268,6 +268,24 @@ rules global@Global{..} args = do
                 cmd "strip -o"
                     [out, releaseBinDir </> binaryName </> stackExeFileName]
 
+    releaseDir </> binaryInstallerFileName %> \out -> do
+        need [releaseDir </> binaryExeFileName]
+        need [releaseDir </> binaryInstallerNSIFileName]
+
+        actionOnException
+            (command_ [Cwd releaseDir] "c:\\Program Files (x86)\\NSIS\\Unicode\\makensis.exe"
+                [ "-V3"
+                , binaryInstallerNSIFileName])
+            (removeFile out)
+
+    releaseDir </> binaryInstallerNSIFileName %> \out -> do
```

Comment:
I think it'd be best to put the NSI file in a different subdirectory.  Something like `_release/installer/stack-<X.Y.Z>-windows-<ARCH>/stack-installer.nsi` (or `releaseInstallerDir </> binaryName </> binaryInstallerNSIFileName`, after setting `releaseInstallerDir` to `releaseDir </> "installer"`)

Reply:
Oh now I noticed you added the version to `stack-installer.nsi` itself.  That is probably fine.  I'll test it now.